pyrobosim/pyrobosim/utils/pose.py

```diff
@@ -94,7 +122,7 @@ def from_transform(cls, tform):
         )
 
     @classmethod
-    def from_dict(cls, pose_dict):
+    def from_dict(cls, pose_dict, angle_units="radians"):
```

Comment:
I feel like in this case, `angle_units` should actually just be part of the dict, instead of a separate argument.

Although this does make it tricky w.r.t. how the `construct()` method dispatches things.

... but I'd rather drop support for units in the `from_list()` method if it means that the dictionary can contain the units rather than needing a separate argument.

What do you think?

Reply:
We can choose between two approaches, as you mentioned:

1. Only support specifying angles in degrees using the `from_dict()` class method. In this case, we would include `angle_units` as a key in the dictionary, which would make it easier to define the YAML world.

2. Use `angle_units` as a separate argument `only` for `from_list()` constructor. However, for this, we would also need to provide angle_units as additional arg in the `construct` method only for `from_list()` method. The second approach offers more flexibility for defining the `Pose` in degrees if the user doesn't want to use YAML for defining the world.

which one do you think we should move forward with?

Reply:
I prefer 1, I think... it's simpler, and that is important.

Reply:
okay then. I will work on this